### ISSUE TITLE
⛔ 🐞 Bug Report: Convert to Container missing for Sections after updating to Elementor 3.15.0

### ISSUE THREAD

**Issue by carlitossuper1** [elementor/elementor#23280]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

After the 3.15 Elementor Update, the Convert to Container option disappeared for a single Section. It still has the option on the Settings of the Page, but it converts ALL the sections.

### Steps to reproduce

1. Edit a page with elementor.
2. Select a Section
3. On the Layout of the Edit Section tab, the Convert to Container is missing now.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server - GPL v8.0.33
	PHP Version: 8.1.20
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: https://test2.wp.alveus.eu
	Home URL: https://test2.wp.alveus.eu
	WP Multisite: No
	Max Upload Size: 32 MB
	Memory limit: 512M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 1
	Debug Mode: Inactive

== Theme ==
	Name: Alveus 2023
	Version: 1.0
	Author: Iconos Consultores
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.1.7
		Author: WP Engine

	Contact Form 7
		Version: 5.7.7
		Author: Takayuki Miyoshi

	Contact Form 7 Multilingual
		Version: 1.2.1
		Author: OnTheGoSystems

	Duplicate Page
		Version: 4.5.2
		Author: mndpsingh287

	Elementor
		Version: 3.15.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.15.0
		Author: Elementor.com

	FileBird Pro
		Version: 5.5
		Author: Ninja Team

	Login with Google
		Version: 1.3.1
		Author: rtCamp

	Safe SVG
		Version: 2.1.1
		Author: 10up

	WP Mail SMTP
		Version: 3.8.2
		Author: WP Mail SMTP

	WPML Multilingual CMS
		Version: 4.6.4
		Author: OnTheGoSystems

	WPML SEO
		Version: 2.1.0
		Author: OnTheGoSystems

	WPML String Translation
		Version: 3.2.6
		Author: OnTheGoSystems

	Yoast SEO
		Version: 20.12
		Author: Team Yoast


== Elements Usage ==
	
	container : 19
		button : 8
		container : 32
		counter : 1
		heading : 7
		image : 3
		social-icons : 1
		testimonial-carousel : 2
		text-editor : 9
		wpml-language-switcher : 1
	section : 37
		button : 4
		container : 6
		heading : 6
		image : 1
		spacer : 1
		text-editor : 1
	wp-page : 249
		button : 348
		call-to-action : 340
		container : 1453
		counter : 103
		form : 1
		google_maps : 2
		heading : 322
		html : 1
		icon : 1
		icon-box : 8
		image : 520
		image-box : 103
		image-carousel : 1
		nested-carousel : 39
		shortcode : 242
		spacer : 1
		testimonial-carousel : 1
		text-editor : 415
		video : 1
	error-404 : 7
		button : 1
		container : 2
		heading : 2
		text-editor : 1


== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap


== Features ==
	Custom Fonts: 2
	Custom Icons: 1

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Active
	Taxonomy Filter: Inactive by default


== Log ==
	
PHP: showing 11 of 11PHP: 2023-07-11 11:19:44 [warning X 2][../wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/icons/custom-icons.php::199] unlink(font_alveus-icons.css): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-11 11:20:49 [error X 28][../wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/icons/custom-icons.php::229] Uncaught Error: Class "ZipArchive" not found in ../wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/icons/custom-icons.php:229
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/icons/custom-icons.php(279): ElementorPro\Modules\AssetsManager\AssetTypes\Icons\Custom_Icons->extract_zip()
#1 ../wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/icons/custom-icons.php(307): ElementorPro\Modules\AssetsManager\AssetTypes\Icons\Custom_Icons->upload_and_extract_zip()
#2 [internal function]: ElementorPro\Modules\AssetsManager\AssetTypes\Icons\Custom_Icons->custom_icons_upload_handler()
#3 ../wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#4 ../wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#5 ../wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#6 ../wp-includes/plugin.php(517): WP_Hook->do_action()
#7 ../wp-admin/admin-ajax.php(188): do_action()
#8 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-11 11:59:55 [warning X 1][../wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/icons/custom-icons.php::199] unlink(alveus-icons.svg): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-15 18:53:20 [warning X 8][../wp-content/plugins/elementor-pro/modules/forms/classes/ajax-handler.php::76] Undefined array key "form_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-16 00:38:03 [warning X 4][../wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-image.php::40] Undefined array key 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-27 22:21:13 [warning X 5][../wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::265] Attempt to read property "ID" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-31 16:36:47 [notice X 274][../wp-content/plugins/elementor/includes/utils.php::727] strtolower(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-31 21:11:36 [warning X 2][../wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-31 21:11:51 [warning X 2908][../wp-content/plugins/elementor/core/kits/manager.php::323] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-31 21:13:23 [notice X 1140][../wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-31 23:06:51 [notice X 64][../wp-content/plugins/elementor/includes/managers/image.php::81] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 20 of 21JS: 2023-07-12 16:24:28 [error X 23][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/7fada1a61d72da6d46ab.bundle.min.js:2:7826] Cannot read properties of null (reading 'dataset') 
JS: 2023-07-12 17:32:53 [error X 19][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.1:3:688684] this.model.isValidChild is not a function 
JS: 2023-07-12 23:39:02 [error X 3][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/web-cli.min.js?ver=3.14.1:3:107466] Routes: `panel/page-settings/yoast-tab` not found. 
JS: 2023-07-14 00:12:53 [error X 3][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.1:3:54034] Cannot read properties of undefined (reading 'activate') 
JS: 2023-07-14 04:41:28 [error X 4][https://test2.wp.alveus.eu/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.14.1:2:64105] Cannot convert undefined or null to object 
JS: 2023-07-15 03:53:18 [error X 355][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.14.1:2:7603] Cannot read properties of undefined (reading 'contains') 
JS: 2023-07-15 15:46:18 [error X 158][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.14.1:2:11953] Cannot read properties of undefined (reading 'attributes') 
JS: 2023-07-15 15:47:44 [error X 8][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.1:3:640984] Cannot read properties of undefined (reading 'isDesignable') 
JS: 2023-07-15 17:40:30 [error X 60][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.1:3:669031] elementorFrontend.elements.window.jQuery is not a function 
JS: 2023-07-16 02:29:07 [error X 2][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.1:3:932267] Cannot read properties of undefined (reading 'route') 
JS: 2023-07-16 16:25:49 [error X 111][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.14.1:3:7470] Cannot read properties of null (reading 'editor') 
JS: 2023-07-19 03:17:10 [error X 1][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.1:3:602454] Cannot read properties of null (reading 'id') 
JS: 2023-07-19 21:18:10 [error X 1][https://test2.wp.alveus.eu/wp-admin/js/editor.min.js?ver=6.2.2:2:1800] Cannot read properties of undefined (reading 'resizeTo') 
JS: 2023-07-19 23:35:14 [error X 1][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/web-cli.min.js?ver=3.14.1:3:107466] Routes: `panel/editor/categories` not found. 
JS: 2023-07-22 11:28:51 [error X 1][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.1:3:131814] Cannot read properties of undefined (reading 'match') 
JS: 2023-07-24 21:57:50 [error X 2][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.14.1:3:4197] this.children.clear is not a function 
JS: 2023-07-24 21:58:01 [error X 10][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.1:3:618748] Cannot read properties of undefined (reading 'isEditable') 
JS: 2023-07-25 00:40:12 [error X 11][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/lib/swiper/swiper.min.js?ver=5.3.6:13:14331] Maximum call stack size exceeded 
JS: 2023-07-25 21:34:27 [error X 305][https://test2.wp.alveus.eu/wp-includes/js/jquery/jquery.min.js?ver=3.6.4:2:31823] Cannot read properties of undefined (reading 'value') 
JS: 2023-07-26 02:29:06 [error X 12][https://test2.wp.alveus.eu/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.1:3:807522] Cannot read properties of undefined (reading 'localized') 

Log: showing 10 of 102023-07-31 15:36:15 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.1',
  'to' => '3.15.0',
)]
2023-07-31 15:36:15 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.14.1',
  'to' => '3.15.0',
)]
2023-07-31 15:36:26 [info] elementor::elementor_updater Started 
2023-07-31 15:36:27 [info] Elementor/Upgrades - _on_each_version Start  
2023-07-31 15:36:28 [info] Elementor/Upgrades - _on_each_version Finished 
2023-07-31 15:36:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.1',
  'to' => '3.15.0',
)]
2023-07-31 15:36:33 [info] elementor-pro::elementor_pro_updater Started 
2023-07-31 15:36:33 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-07-31 15:36:33 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-07-31 15:36:33 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.14.1',
  'to' => '3.15.0',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by nicholaszein** [comment]
Hello @carlitossuper1,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
We tested using the latest version of Elementor and the option to `Convert to container` is available as expected. See:

![image](https://github.com/elementor/elementor/assets/113993460/34e461c5-be22-40b9-a8b0-74a125289ffb)

----

Your issue is most likely caused by a conflict with another plugin or your theme.

### ✅ What you can do to solve your issue:
Try these **Troubleshooting steps**:

#### ⚠️ Make a Backup of your website before proceeding, and if possible do it in a staging environment!

1. 🔬 **Isolate the problem:** Deactivate all of your plugins (except Elementor/Pro), and temporarily change your WordPress theme to [Hello theme](https://wordpress.org/themes/hello-elementor/). If this solves the problem, gradually activate your plugins and theme one by one, until you spot the conflicting plugin or theme.

2. 🗑️ **Clear the cache:** Clear all levels of cache, including but not limited to **Server cache**, **WordPress (optimization plugin) cache**, **CDN cache**, and **Browser cache**. Also, try visiting the front page of your website in **incognito/private** mode.

3. 📋 **Meet the Requirements:** Make sure the server resources allocated to your site and WordPress installation meet our [Requirements](https://elementor.com/help/requirements/). 
    - To check this information go to your WordPress `Dashboard` > `Elementor` > `System info` [learn how to copy it here](https://elementor.com/help/hare-my-system-info/). We can't accept issues without it.

### 👎 If the issue persists use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

#### 👍 For now, that's all we can help with. 

### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

#### ⚠️ As a gentle reminder, issues that do not follow our 🐞 Bug Report Guidelines may need to be closed without troubleshooting. 

Remember to always follow our **[🐞 Bug Report Guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&projects=&template=bug.yml)** to save time and ensure smooth troubleshooting.

Thank you for understanding! 🙏

Kind regards
